packages/web/pages/onboarding.tsx

```diff
@@ -0,0 +1,184 @@
+import { Box, Tooltip, VStack } from '@metafam/ds';
+import { PageContainer } from 'components/Container';
+import { LandingHeader } from 'components/Landing/LandingHeader';
+import { Onboard } from 'components/Landing/Onboard';
+import { MetaLink } from 'components/Link';
+import { HeadComponent } from 'components/Seo';
+// import { gsap } from "gsap";
+// import { ScrollToPlugin } from 'gsap/dist/ScrollToPlugin';
+// import { ScrollTrigger } from 'gsap/dist/ScrollTrigger';
+// import { useRouter } from 'next/router';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import {
+  FaDiscord,
+  FaGithub,
+  FaHome,
+  FaTrophy,
+  FaTwitter,
+  FaUserCircle,
+} from 'react-icons/fa';
+
+export const getStaticProps = async () => ({
+  props: {
+    hideTopMenu: true,
+  },
+});
+
+const Onboarding: React.FC = () => {
+  const scrollContainer =
+    typeof document !== 'undefined'
+      ? document.getElementById('scroll-container')
+      : null;
+  const [section, setSection] = useState(0);
+  const hostName = useRef('https://metagame.wtf');
```

Comment:
Can `hostName` be stored as a constant outside the component ?

Reply:
sure. will do.